src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -1011,7 +1011,6 @@ $dark-focusColor: var(--green-500, color(green-500));
   position: absolute;
   top: var(--spacing-xs, spacing(xs));
   right: var(--spacing-md, spacing(md));
-  z-index: 2;
```

Comment:
I don't think this should be removed; instead the `.k-modal-fullscreen-header` likely just needs a higher `z-index`

Does adding `z-index` there cause issues?

Reply:
What's the reasoning behind `z-index: 2` here? I checked the stack order and the tab sequence and its same without it.

Reply:
TBH I'm not 100%. @kleinfreund created the component; I'm guessing it's to keep the copy button over the code.

What if we just drop it to `1` instead? I still think we should bump the z-index of the modal header

Reply:
Updated.